src/box/lua/net_box.lua

```diff
@@ -1233,8 +1213,7 @@ this_module = {
     new = connect, -- Tarantool < 1.7.1 compatibility,
     _method = { -- for tests
         ping        = M_PING,
-        call_16     = M_CALL_16,
-        call_17     = M_CALL_17,
+        call_17     = M_CALL,
```

Comment:
Nit: Please rename this to `call` as well.

Reply:
Fixed.